src/main/java/io/jenkins/plugins/orka/client/ConfigurationRequest.java

```diff
@@ -32,7 +32,7 @@ public class ConfigurationRequest {
     private String scheduler;
 
     @SuppressFBWarnings("URF_UNREAD_FIELD")
-    private int memory;
+    private Float memory;
```

Comment:
is there a reason not to use the value type here (`float` instead of `Float`)?

Reply:
No... I wasn't aware that there was both `float` and `Float` in Java

Reply:
@DanMHammer can we switch to the value type then? We don't need to sandbox/unsandbox all the time